project-ideas/detailed-specs/cypress-testing.md

```diff
@@ -88,6 +88,33 @@ retry by rerunning the second command above.
 you'll have to run the same *start* command again to build it from scratch. You will need to have Biotestmine running for the steps
 below to succeed.
 
+**Note:** Incase you run into issues during the build under building intermine_builder  ``` RUN cpan -i App::cpanminus ```  you are advised to make edits to
```

Comment:
Could you mention in this paragraph that the user will need to clone the intermine_boot repository? They may be confused where to find that file.

Reply:
Alright thanks for the feedback, let me work on it